src/routes/role/data.ts

```diff
@@ -0,0 +1,58 @@
+import { DBTables, generateBaseDBfields } from '../../constants/db.ts';
+import type { CreateRoleData, Role, UpdateRoleData } from './validation.ts';
+
+export async function insertRole(database: D1Database, data: CreateRoleData) {
+	const baseDbfields = generateBaseDBfields();
+
+	const { success } = await database.prepare(`
```

Comment:
This is using an older pattern that is less readable than the newest one:

https://github.com/torontojs/vms/blob/dev/src/routes/profile/data.ts